.github/workflows/gpu_test.yml

```
@@ -0,0 +1,48 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Test V3
+
+on:
+  push:
+    branches: [ v3 ]
+  pull_request:
+    branches: [ v3 ]
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  test:
+    name: py=${{ matrix.python-version }}, np=${{ matrix.numpy-version }}, deps=${{ matrix.dependency-set }}
+
+    runs-on: "cirun-gpu-runner--${{ github.run_id }}"
```

```suggestion
    runs-on: "gpu-runner"
```

I am not 100% sure but I think changing to this will have this action using the github actions GPU runner that @rabernat and I just configured.

---

Just reverted the cirun commits and added a github CI workflow for GPU tests. Also tried to make the GPU workflow only run GPU tests. Can certainly optimize the test matrix more for cost.